downstream/modules/platform/proc-gw-organizations-exec-env.adoc

```diff
@@ -2,24 +2,24 @@
 
 [id="proc-gw-organizations-exec-env_{context}"]
 
-= Working with execution environments
+= Working with {ExecEnvShort}s
 
-When automation controller is enabled on the platform, you can review any execution environments you have set up and manage their settings within the organization resource.
+When {ControllerName} is enabled on the platform, you can review any {ExecEnvShort}s you have set up and manage their settings within the organization resource.
 
 For more information about execution environments, see link:{BaseURL}/{PlatformVers}/html/using_automation_execution/index#assembly-controller-execution-environments[Execution environments] in _Using automation execution_ guide.
 
 
 .Procedure
 
 . From the navigation panel, select {MenuAMOrganizations}.
-. From the Organizations list view, select the organization to which you want to manage execution environments.
+. From the Organizations list view, select the organization to which you want to manage {ExecEnvShort}s.
```

Comment:
I'd change to "select the organization whose EEs you want to manage."

Reply:
Fixed.